CoreFoundation/RunLoop.subproj/CFRunLoop.c

```diff
@@ -411,7 +422,7 @@ static __CFPortSet __CFPortSetAllocate(void) {
     result->used = 0;
     result->size = 4;
     result->handles = (HANDLE *)CFAllocatorAllocate(kCFAllocatorSystemDefault, result->size * sizeof(HANDLE), 0);
-    CF_SPINLOCK_INIT_FOR_STRUCTS(result->lock);
+    CF_LOCK_INIT_FOR_STRUCTS(result->lock);
```

Comment:
I am not sure we can change this lock type here btw.

Reply:
L415 shows that this is of type `CFLock_t`.